AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -18,6 +18,7 @@ amp-reddit-com.cdn.ampproject.org##.AppSelectorModal__body
 reddit.com#$#.XPromoPopup { display: none!important; }
 reddit.com#$#body.scroll-disabled { overflow: visible!important; position: static!important; }
 !
+pikicast.com##.app-story-layer
```

Comment:
we also need to remove the overlay and fix scrolling

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/82955/commits/fb8d6ca0c67396fea97cb70dec4e2266bd0bdca3